test/example/echo/echo.yaml

```diff
@@ -2,34 +2,54 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: echoheaders
+  name: service1
   labels:
-    app: echoheaders
+    app: service1
 spec:
   type: NodePort
   ports:
   - port: 80
     targetPort: 8080
     name: http
   selector:
-    app: echoheaders
+    app: service1
 ---
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: echoheaders
+  name: service1
 spec:
   replicas: 3
   template:
     metadata:
       labels:
-        app: echoheaders
+        app: service1
+      annotations:
+        pod.beta.kubernetes.io/init-containers: '[
+          {
+            "name": "iptables",
+            "image": "gcr.io/istio-test/init:test",
+            "imagePullPolicy": "Always",
+            "securityContext": {
+              "privileged": true
+            },
+            "command": ["/bin/bash", "-c", "sudo iptables -t nat -A PREROUTING -p tcp --dport 8080 -j REDIRECT --to-port 9999"]
+          }
+        ]'
     spec:
       containers:
-      - name: echoheaders
+      - name: echo
         image: gcr.io/google_containers/echoserver:1.4
         ports:
         - containerPort: 8080
+      - name: proxy
+        image: gcr.io/istio-test/runtime:test
+        args: ["proxy", "-s", "manager:8080", "--logtostderr", "-v", "2", "-p", "9999"]
+        securityContext:
+          privileged: true
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 9999
```

Comment:
What about init containers? Is it OK to run privileged mode commands (`iptables`) in the init container?
@enricoschiattarella @rshriram The proxy doesn't need extra privileges. That was just a leftover from experimentation.